zqd/middleware.go

```diff
@@ -76,9 +76,9 @@ func panicCatchMiddleware(logger *zap.Logger) mux.MiddlewareFunc {
 				if r == nil {
 					return
 				}
-				rstr := fmt.Sprint(r)
-				logger.DPanic("Panic", zap.String("error", rstr))
-				http.Error(w, rstr, http.StatusInternalServerError)
+				err := zqe.RecoverError(r)
+				logger.DPanic("panic", zap.Error(err))
+				http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
```suggestion
				logger.DPanic("Panic", zap.String("error", fmt.Sprint(r)))
				err := zqe.RecoverError(r)
				http.Error(w, err.Error(), http.StatusInternalServerError)
```
since
1. we usually capitalize the first word of the log message and
2. zap already includes the stack trace at DpanicLevel.